### PR TITLE
April release notes

### PR DESCRIPTION
**Pull request by yihuiliao**
Closes 

## ✅ Pull Request Checklist:

- [ ] Included link to corresponding [React Spectrum GitHub Issue](https://github.com/adobe/react-spectrum/issues).
- [ ] Added/updated unit tests and storybook for this change (for new code or code which already has tests).
- [ ] Filled out test instructions.
- [ ] Updated documentation (if it already exists for this component).
- [ ] Looked at the Accessibility Practices for this feature - [Aria Practices](https://www.w3.org/WAI/ARIA/apg/)

### CODE REVIEW

File: packages/dev/docs/pages/releases/2024-04-25.mdx [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+* Support `onAction` and `isDisabled` on item elements - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/5874)
+* Add conditional drag and drop support - [@ryo-manba](https://github.com/ryo-manba) - [PR](https://github.com/adobe/react-spectrum/pull/5641)
+* Add `validationBehavior` to Form - [@reidbarber](https://github.com/reidbarber) - [PR](https://github.com/adobe/react-spectrum/pull/6062)
+* Add support for `routerOptions` and `useHref` - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/5864)
+* Add support for `shouldFlip` in RAC Tooltip - [@ktabors](https://github.com/ktabors) - [PR](https://github.com/adobe/react-spectrum/pull/6147)
+* Turn off virtualizer when running in test env - [@LFDanLu](https://github.com/LFDanLu) - [PR](https://github.com/adobe/react-spectrum/pull/5192)
+* Add support to display all toasts - [@ktabors](https://github.com/ktabors) - [PR](https://github.com/adobe/react-spectrum/pull/5933)
+* S1 style macro - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/6009)
```
Comment (LFDanLu) [inline packages/dev/docs/pages/releases/2024-04-25.mdx:39]: I'm not sure if we are going to expose this macro stuff, that package might remain internal?

File: packages/dev/docs/pages/releases/2024-04-25.mdx [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+* Update cursor style to default for Tabs in docs - [@ryo-manba](https://github.com/ryo-manba) - [PR](https://github.com/adobe/react-spectrum/pull/5982)
+* Add ContextualHelp example for disabled button to Tooltip docs - [@reidbarber](https://github.com/reidbarber) - [PR](https://github.com/adobe/react-spectrum/pull/6018)
+* Fix the controlled RadioGroup example - [@sookmax](https://github.com/sookmax) - [PR](https://github.com/adobe/react-spectrum/pull/6057)
+* Fix various typos in docs - [@reidbarber](https://github.com/reidbarber) - [PR](https://github.com/adobe/react-spectrum/pull/5869)
+* Fix typo in `useListData` docs - [@ryo-manba](https://github.com/ryo-manba) - [PR](https://github.com/adobe/react-spectrum/pull/6106)
+* Escape `<form>` in `ButtonProps` with backticks - [@sookmax](https://github.com/sookmax) - [PR](https://github.com/adobe/react-spectrum/pull/6041)
+* Update testing documentation and expose some test utils - [@LFDanLu](https://github.com/LFDanLu) - [PR](https://github.com/adobe/react-spectrum/pull/5891)
+
+## Under Construction
```
Comment (LFDanLu) [inline packages/dev/docs/pages/releases/2024-04-25.mdx:99]: Self reminder to check/update these version values + add TreeView. Probably move submenu out of this section if it is hitting GA

**Review by LFDanLu — COMMENTED** [review]
know this is a draft and will be updated, just a couple of reminders + a comment

File: packages/dev/docs/pages/releases/2024-04-25.mdx [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+* Turn off virtualizer when running in test env - [@LFDanLu](https://github.com/LFDanLu) - [PR](https://github.com/adobe/react-spectrum/pull/5192)
+* Add support to display all toasts - [@ktabors](https://github.com/ktabors) - [PR](https://github.com/adobe/react-spectrum/pull/5933)
+* S1 style macro - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/6009)
+* Allow passing of `inputRef` to RAC Checkbox/Switch - [@yihuiliao](https://github.com/yihuiliao) - [PR](https://github.com/adobe/react-spectrum/pull/5967)
+* Allow passing of `inputRef` to RAC Radio - [@sookmax](https://github.com/sookmax) - [PR](https://github.com/adobe/react-spectrum/pull/6067)
+* Simplify color naming for accessibility - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/6180)
+* Improve color naming thresholds - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/6210)
+* Give translators more flexibility with transparent color names - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/6238)
```
Comment (snowystinger) [inline packages/dev/docs/pages/releases/2024-04-25.mdx:44]: these three were all part of one, we should just condense them under one bullet

File: packages/dev/docs/pages/releases/2024-04-25.mdx [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+* Add `validationBehavior` to Form - [@reidbarber](https://github.com/reidbarber) - [PR](https://github.com/adobe/react-spectrum/pull/6062)
+* Add support for `routerOptions` and `useHref` - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/5864)
+* Add support for `shouldFlip` in RAC Tooltip - [@ktabors](https://github.com/ktabors) - [PR](https://github.com/adobe/react-spectrum/pull/6147)
+* Turn off virtualizer when running in test env - [@LFDanLu](https://github.com/LFDanLu) - [PR](https://github.com/adobe/react-spectrum/pull/5192)
+* Add support to display all toasts - [@ktabors](https://github.com/ktabors) - [PR](https://github.com/adobe/react-spectrum/pull/5933)
+* S1 style macro - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/6009)
+* Allow passing of `inputRef` to RAC Checkbox/Switch - [@yihuiliao](https://github.com/yihuiliao) - [PR](https://github.com/adobe/react-spectrum/pull/5967)
+* Allow passing of `inputRef` to RAC Radio - [@sookmax](https://github.com/sookmax) - [PR](https://github.com/adobe/react-spectrum/pull/6067)
```
Comment (snowystinger) [inline packages/dev/docs/pages/releases/2024-04-25.mdx:41]: these two can be condensed

File: packages/dev/docs/pages/releases/2024-04-25.mdx [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+* Fix `useRangeCalendar` with mobile scrolling - [@QzCurious](https://github.com/QzCurious) - [PR](https://github.com/adobe/react-spectrum/pull/5976)
+* Fix TypeError in ListBox's grid layout keyboard navigation - [@sookmax](https://github.com/sookmax) - [PR](https://github.com/adobe/react-spectrum/pull/6172)
+* Remove italics from ComboBox and SearchAutocomplete - [@yihuiliao](https://github.com/yihuiliao) - [PR](https://github.com/adobe/react-spectrum/pull/6190)
+* Automatically set `MenuItem` `textValue` from children if it is a string - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/5868)
+* Bring back i18n string parcel config - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/5889)
+* Omit `className` and `style` prop in RSP DropZone - [@yihuiliao](https://github.com/yihuiliao) - [PR](https://github.com/adobe/react-spectrum/pull/5961)
+* Expose `shouldCloseOnInteractOutsid`' on Modal/ModalOverlay - [@sookmax](https://github.com/sookmax) - [PR](https://github.com/adobe/react-spectrum/pull/5986)
```
Comment (snowystinger) [inline packages/dev/docs/pages/releases/2024-04-25.mdx:74]:
```suggestion
* Expose `shouldCloseOnInteractOutside`' on Modal/ModalOverlay - [@sookmax](https://github.com/sookmax) - [PR](https://github.com/adobe/react-spectrum/pull/5986)
```

File: packages/dev/docs/pages/releases/2024-04-25.mdx (hunk 1 above)
Comment (yihuiliao) [reply packages/dev/docs/pages/releases/2024-04-25.mdx:39]: ahh okay good to know - i'll just take it out for now

File: packages/dev/docs/pages/releases/2024-04-25.mdx (hunk 2 above)
Comment (yihuiliao) [reply packages/dev/docs/pages/releases/2024-04-25.mdx:99]: yeah wasn't sure about the versions so i just left them here in this section. i'll keep it here for now but move it once we are sure

**Comment by yihuiliao** [comment]
i've intentionally left out the UNSAFE_portal stuff since I figured we didn't want that to be public but let me know if we should include it

File: packages/dev/docs/pages/releases/2024-05-01.mdx [hunk 6]
```diff
@@ -0,0 +1,156 @@
...
+- TreeView - [@LFDanLu](https://github.com/LFDanLu) - [PR](https://github.com/adobe/react-spectrum/pull/6020)
+
+**Color Components (Beta):**
+
+- Add color naming for accessibility - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/6180), [PR](https://github.com/adobe/react-spectrum/pull/6210), [PR](https://github.com/adobe/react-spectrum/pull/6238)
+- Add ColorArea, ColorField, ColorSlider, and ColorWheel to RAC - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/6199), [@ktabors](https://github.com/ktabors) - [PR](https://github.com/adobe/react-spectrum/pull/6278)
+- Add ColorSwatch component - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/6204)
+- Add ColorPicker and ColorSwatchPicker components - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/6272)
+
+## Released packages
```
Comment (devongovett) [inline packages/dev/docs/pages/releases/2024-05-01.mdx:156]:
```
 - @adobe/react-spectrum@3.35.0
 - @internationalized/date@3.5.3
 - @internationalized/message@3.1.3
 - @internationalized/number@3.5.2
 - @internationalized/string-compiler@3.2.3
 - @internationalized/string@3.2.2
 - @react-aria/accordion@3.0.0-alpha.28
 - @react-aria/actiongroup@3.7.4
 - @react-aria/aria-modal-polyfill@3.7.9
 - @react-aria/autocomplete@3.0.0-alpha.28
 - @react-aria/breadcrumbs@3.5.12
 - @react-aria/button@3.9.4
 - @react-aria/calendar@3.5.7
 - @react-aria/checkbox@3.14.2
 - @react-aria/color@3.0.0-beta.32
 - @react-aria/combobox@3.9.0
 - @react-aria/datepicker@3.10.0
 - @react-aria/dialog@3.5.13
 - @react-aria/dnd@3.6.0
 - @react-aria/example-theme@1.0.2
 - @react-aria/focus@3.17.0
 - @react-aria/form@3.0.4
 - @react-aria/grid@3.9.0
 - @react-aria/gridlist@3.8.0
 - @react-aria/i18n@3.11.0
 - @react-aria/interactions@3.21.2
 - @react-aria/label@3.7.7
 - @react-aria/landmark@3.0.0-beta.11
 - @react-aria/link@3.7.0
 - @react-aria/listbox@3.12.0
 - @react-aria/live-announcer@3.3.3
 - @react-aria/menu@3.14.0
 - @react-aria/meter@3.4.12
 - @react-aria/numberfield@3.11.2
 - @react-aria/overlays@3.22.0
 - @react-aria/progress@3.4.12
 - @react-aria/radio@3.10.3
 - @react-aria/searchfield@3.7.4
 - @react-aria/select@3.14.4
 - @react-aria/selection@3.18.0
 - @react-aria/separator@3.3.12
 - @react-aria/slider@3.7.7
 - @react-aria/spinbutton@3.6.4
 - @react-aria/ssr@3.9.3
 - @react-aria/steplist@3.0.0-alpha.4
 - @react-aria/switch@3.6.3
 - @react-aria/table@3.14.0
 - @react-aria/tabs@3.9.0
 - @react-aria/tag@3.4.0
 - @react-aria/test-utils@1.0.0-alpha.0
 - @react-aria/textfield@3.14.4
 - @react-aria/toast@3.0.0-beta.11
 - @react-aria/toggle@3.10.3
 - @react-aria/toolbar@3.0.0-beta.4
 - @react-aria/tooltip@3.7.3
 - @react-aria/tree@3.0.0-alpha.0
 - @react-aria/utils@3.24.0
 - @react-aria/virtualizer@3.10.0
 - @react-aria/visually-hidden@3.8.11
 - @react-spectrum/accordion@3.0.0-alpha.30
 - @react-spectrum/actionbar@3.4.4
 - @react-spectrum/actiongroup@3.10.4
 - @react-spectrum/autocomplete@3.0.0-alpha.31
 - @react-spectrum/avatar@3.0.11
 - @react-spectrum/badge@3.1.12
 - @react-spectrum/breadcrumbs@3.9.6
 - @react-spectrum/button@3.16.3
 - @react-spectrum/buttongroup@3.6.12
 - @react-spectrum/calendar@3.4.8
 - @react-spectrum/card@3.0.0-alpha.32
 - @react-spectrum/checkbox@3.9.5
 - @react-spectrum/color@3.0.0-beta.33
 - @react-spectrum/combobox@3.12.4
 - @react-spectrum/contextualhelp@3.6.10
 - @react-spectrum/datepicker@3.9.5
 - @react-spectrum/dialog@3.8.10
 - @react-spectrum/divider@3.5.12
 - @react-spectrum/dnd@3.3.9
 - @react-spectrum/dropzone@3.0.0
 - @react-spectrum/filetrigger@3.0.0
 - @react-spectrum/form@3.7.5
 - @react-spectrum/icon@3.7.12
 - @react-spectrum/illustratedmessage@3.5.0
 - @react-spectrum/image@3.5.0
 - @react-spectrum/inlinealert@3.2.4
 - @react-spectrum/label@3.16.5
 - @react-spectrum/labeledvalue@3.1.13
 - @react-spectrum/layout@3.6.4
 - @react-spectrum/link@3.6.6
 - @react-spectrum/list@3.7.9
 - @react-spectrum/listbox@3.12.8
 - @react-spectrum/menu@3.19.0
 - @react-spectrum/meter@3.5.0
 - @react-spectrum/numberfield@3.9.2
 - @react-spectrum/overlays@5.6.0
 - @react-spectrum/picker@3.14.4
 - @react-spectrum/progress@3.7.6
 - @react-spectrum/provider@3.9.6
 - @react-spectrum/radio@3.7.5
 - @react-spectrum/searchfield@3.8.5
 - @react-spectrum/slider@3.6.8
 - @react-spectrum/statuslight@3.5.12
 - @react-spectrum/steplist@3.0.0-alpha.4
 - @react-spectrum/switch@3.5.4
 - @react-spectrum/table@3.12.9
 - @react-spectrum/tabs@3.8.9
 - @react-spectrum/tag@3.2.5
 - @react-spectrum/test-utils@1.0.0-alpha.0
 - @react-spectrum/text@3.5.4
 - @react-spectrum/textfield@3.12.0
 - @react-spectrum/theme-dark@3.5.9
 - @react-spectrum/theme-default@3.5.9
 - @react-spectrum/theme-express@3.0.0-alpha.11
 - @react-spectrum/theme-light@3.4.9
 - @react-spectrum/toast@3.0.0-beta.11
 - @react-spectrum/tooltip@3.6.6
 - @react-spectrum/tree@3.0.0-alpha.0
 - @react-spectrum/utils@3.11.6
 - @react-spectrum/view@3.6.9
 - @react-spectrum/well@3.4.12
 - @react-stately/calendar@3.5.0
 - @react-stately/checkbox@3.6.4
 - @react-stately/collections@3.10.6
 - @react-stately/color@3.6.0
 - @react-stately/combobox@3.8.3
 - @react-stately/data@3.11.3
 - @react-stately/datepicker@3.9.3
 - @react-stately/dnd@3.3.0
 - @react-stately/flags@3.0.2
 - @react-stately/form@3.0.2
 - @react-stately/grid@3.8.6
 - @react-stately/layout@3.13.8
 - @react-stately/list@3.10.4
 - @react-stately/menu@3.7.0
 - @react-stately/numberfield@3.9.2
 - @react-stately/overlays@3.6.6
 - @react-stately/radio@3.10.3
 - @react-stately/searchfield@3.5.2
 - @react-stately/select@3.6.3
 - @react-stately/selection@3.15.0
 - @react-stately/slider@3.5.3
 - @react-stately/steplist@3.0.0-alpha.4
 - @react-stately/table@3.11.7
 - @react-stately/tabs@3.6.5
 - @react-stately/toast@3.0.0-beta.3
 - @react-stately/toggle@3.7.3
 - @react-stately/tooltip@3.4.8
 - @react-stately/tree@3.8.0
 - @react-stately/utils@3.10.0
 - @react-stately/virtualizer@3.7.0
 - @react-types/accordion@3.0.0-alpha.20
 - @react-types/actionbar@3.1.6
 - @react-types/actiongroup@3.4.8
 - @react-types/autocomplete@3.0.0-alpha.21
 - @react-types/avatar@3.0.6
 - @react-types/badge@3.1.8
 - @react-types/breadcrumbs@3.7.4
 - @react-types/button@3.9.3
 - @react-types/buttongroup@3.3.8
 - @react-types/calendar@3.4.5
 - @react-types/card@3.0.0-alpha.25
 - @react-types/checkbox@3.8.0
 - @react-types/color@3.0.0-beta.24
 - @react-types/combobox@3.11.0
 - @react-types/contextualhelp@3.2.9
 - @react-types/datepicker@3.7.3
 - @react-types/dialog@3.5.9
 - @react-types/divider@3.3.8
 - @react-types/form@3.7.3
 - @react-types/grid@3.2.5
 - @react-types/illustratedmessage@3.3.8
 - @react-types/image@3.4.0
 - @react-types/label@3.9.2
 - @react-types/layout@3.3.14
 - @react-types/link@3.5.4
 - @react-types/list@3.2.16
 - @react-types/listbox@3.4.8
 - @react-types/menu@3.9.8
 - @react-types/meter@3.4.0
 - @react-types/numberfield@3.8.2
 - @react-types/overlays@3.8.6
 - @react-types/progress@3.5.3
 - @react-types/provider@3.8.0
 - @react-types/radio@3.8.0
 - @react-types/searchfield@3.5.4
 - @react-types/select@3.9.3
 - @react-types/shared@3.23.0
 - @react-types/slider@3.7.2
 - @react-types/statuslight@3.3.8
 - @react-types/switch@3.5.2
 - @react-types/table@3.9.4
 - @react-types/tabs@3.3.6
 - @react-types/text@3.3.8
 - @react-types/textfield@3.9.2
 - @react-types/tooltip@3.4.8
 - @react-types/view@3.4.8
 - @react-types/well@3.3.8
 - @spectrum-icons/color@3.5.12
 - @spectrum-icons/express@3.0.0-alpha.16
 - @spectrum-icons/illustrations@3.6.12
 - @spectrum-icons/ui@3.6.6
 - @spectrum-icons/workflow@4.2.11
 - @react-aria/optimize-locales-plugin@1.1.0
 - @react-aria/parcel-resolver-optimize-locales@1.1.0
 - react-aria@3.33.0
 - react-aria-components@1.2.0
 - react-stately@3.31.0
 - tailwindcss-react-aria-components@1.1.2
```

File: packages/dev/docs/pages/releases/2023-12-20.mdx [hunk 7]
```diff
@@ -26,7 +26,7 @@ As always, huge thanks to everyone in our community for their feedback and contr
 Happy coding and see you in the new year! ☃️
 
 
-### React Aria Components API updates
+** React Aria Components API updates **
```
Comment (devongovett) [inline packages/dev/docs/pages/releases/2023-12-20.mdx:29]:
```suggestion
**React Aria Components API updates**
```

**Review by devongovett — APPROVED** [review]

**Review by dannify — APPROVED** [review]

**Comment by KnisterPeter** [comment]
Hi @devongovett, any reason the release notes are not published to https://react-spectrum.adobe.com/releases/index.html anymore?

**Comment by LFDanLu** [comment]
@KnisterPeter They are showing up here for me: https://react-spectrum.adobe.com/releases/2024-05-01.html, does that link work for you?

**Pull request merged**